### ISSUE TITLE
macOS ShadowsocksX-NG is outdated, newer version can't read the qr code

### ISSUE THREAD

**Issue by hermosabeach** [StreisandEffect/streisand#1613]
The mirrored MacOS Shadowsocks client is outdated.  Current Version of ShadowsocksX-NG is 1.8.2, current Streisand install is ShadowsocksX-NG 1.6.1 (https://github.com/shadowsocks/ShadowsocksX-NG/releases/)

The ShadowsocksX-NG 1.8.2 version cannot read the QR Codes generated by the Streisand install documentation because they do not implement the newer Shadowsocks SIP002 URI Scheme.
https://shadowsocks.org/en/spec/SIP002-URI-Scheme.html
https://shadowsocks.org/assets/whitepaper.pdf

It would be helpful if the generated documentation also displays the URI (ss://...) since some shadowsocks clients (Jigsaw's Outline) do not read QR Codes.

**Comment by lucasff** [comment]
I also agree that having a simple line with the `ss://` scheme is very helpful. Example: you're on Android, how to import if you can't scan your own phone? As a link also the `ss://` scheme can be handled directly by the app (used on other websites and it works nice)
